packages/components/src/components/select/select.stories.ts

```diff
@@ -71,600 +57,266 @@ export default {
 
 export const Default = {
   render: (args: any) => {
-    return html`<div class="h-[260px] w-[420px]">${generateTemplate({ args })}</div>`;
+    return html`<div class="h-[260px] w-[500px]">${generateTemplate({ args })}</div>`;
   }
 };
 
 /**
- * Use the `size` attribute to change a select’s size. It will cascade to slotted `sd-option` elements.
+ * Use the `size` attribute the size. It will cascade to slotted `sd-option` elements.
  */
 
-export const SizeMultiple = {
-  name: 'Size x Multiple',
-  parameters: {
-    controls: {
-      exclude: ['open-attr']
-    }
-  },
-  render: (args: { 'open-attr'?: string }) => {
-    delete args['open-attr'];
-
-    return html`<div class="h-[340px]">
-      ${generateTemplate({
-        options: {
-          classes: 'w-full'
-        },
-        axis: {
-          x: {
-            type: 'attribute',
-            name: 'size'
-          },
-          y: {
-            type: 'attribute',
-            name: 'useTags',
-            values: [false, true]
-          }
-        },
-        constants: [
-          fiveOptionsConstant,
-          multipleConstant,
-          { type: 'attribute', name: 'value', value: 'option-1 option-2 option-3 option-4' }
-        ],
-        args
-      })}
-    </div>`;
-  }
+export const Size = {
+  render: () => html`
```

Comment:
Could we show it like that? I used the placeholder text to display the used size.

![image](https://github.com/user-attachments/assets/c3c1b5ef-dd03-4833-8da0-2b4393c7f743)

Reply:
Size of the options is fixed.

Unfortunately making the selects open by default makes the page scroll down to this story because each element gets the focus state as if they had been clicked. Also it feels a bit buggy because they don't show up always open. Although I see the value in it,  I would recommend not to display the opened version.

Reply:
@smfonseca please make sure you use the same copy in the samples.
Also in the last one instead os large should same "Small" (i have updated in figma)

Reply:
Updated